lapce-data/src/proxy.rs

```diff
@@ -400,6 +401,12 @@ impl LapceProxy {
             .send_rpc_notification("install_plugin", &json!({ "plugin": plugin }));
     }
 
+    /// Broadcast an event to plugins which are subscribed to it
+    pub fn plugin_broadcast(&self, event: PluginEvent) {
```

Comment:
Another thought on this. Between the communication between UI and Proxy, we probably don't make them plugin specific. E.g., ```new_buffer```, ```save_buffer``` would be plugin events at Proxy's layer.

Reply:
While I agree some events shouldn't be plugin specific, it still seems useful to be able to only send an event to a certain plugin. Ex: A plugin registers a workbench command. Then the proxy/whatever can specifically send the "workbench command used" event just to the plugin that registered it. I think I might be misunderstanding what you mean, though.